Filters/exclusions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/185
+||adjust.com^
```

Comment:
That's too much, we shouldn't exclude EVERYTHING from adjust, only app.adjust

Reply:
ok, changed to `@@||app.adjust.com^|` (`||adjust.com^$third-party` is in Easylist)